integration-tests/tsconfig.json

```diff
@@ -23,5 +23,5 @@
     "baseUrl": "."
   },
   "exclude": ["node_modules"],
-  "include": ["./**/*.ts"]
+  "include": ["./**/*.ts", "../client/UserResolver.test.ts"]
```

Comment:
Le chemin ./**/*.ts ne prend pas en compte se fichier de base ?

Reply:
You absolutly right, I removed it